hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumes.java

```diff
@@ -146,7 +146,6 @@ private void setVolumeAcl(ObjectStore objectStore, String volumeName,
   private void checkUser(MiniOzoneCluster cluster, UserGroupInformation user,
       List<String> expectVol, boolean expectListAllSuccess) throws IOException {
 
-    UserGroupInformation.setLoginUser(user);
```

Comment:
Just a note:

Removing this line enables the existing tests to detect the bug solved by this jira.
You see previously I set the login user to the user I'm calling listVolume with, which unintentionally hides the bug.

Reply:
I've also added new test cases.